index.ts

```diff
@@ -11,30 +11,48 @@ import { klona } from "klona/json";
 import is from "@slimio/is";
 import Locker from "@slimio/lock";
 import isMinified from "is-minified-code";
-import JSXRay from "js-x-ray";
+import JSXRay from "@nodesecure/js-x-ray";
 
 // Import Internal Dependencies
-import { fetchPackage, getTarballComposition } from "./src/utils.js";
+import { fetchPackage, getTarballComposition } from "./src/utils";
```

Comment:
Because we are compiling to ESM you need to use `.js` extension here (that mandatory even with TypeScript).

`import { fetchPackage, getTarballComposition } from "./src/utils.js";`